CHANGELOG.md

```diff
@@ -5,6 +5,7 @@ All notable changes to this project will be documented in this file.
 
 ### Added
 
+-   #2025 : Optimise min/max to avoid temp variables when exist
```

Comment:
Can this be moved to the "Fixed" section. I also suggest a minor grammar improvement:
```suggestion
### Added

-   #2025 : Optimise min/max to avoid unnecessary temporary variables.
```

Reply:
Can this be moved to the "Fixed" section

Reply:
@EmilyBourne yes, moved it to `Fixed` section.